build.sbt

```diff
@@ -261,8 +260,6 @@ def updateWebsiteTag =
 
 lazy val commonSettings = ReleasePlugin.extraReleaseCommands ++ Seq(
   organization := "io.getquill",
-  scalaVersion := "2.11.12",
-  crossScalaVersions := Seq("2.11.12","2.12.6"),
```

Comment:
@fwbrasil I've figured out that those properties are useless and ignored by our build/publish scripts, hence removed that. 
Besides by default, sbt sets scala version to 2.12.6 so it will be also applicable for developers

Reply:
that didn't help, will try to fix tomorrow

Reply:
I think it works if you forward the system property from the bash script usin `-D`